specification/digitaltwins/resource-manager/Microsoft.DigitalTwins/stable/2022-10-31/digitaltwins.json

```diff
@@ -0,0 +1,2299 @@
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "DigitalTwins instance properties.",
+          "$ref": "#/definitions/DigitalTwinsProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/DigitalTwinsResource"
+        }
+      ]
+    },
+    "DigitalTwinsPatchDescription": {
+      "description": "The description of the DigitalTwins service.",
+      "type": "object",
+      "properties": {
+        "tags": {
+          "description": "Instance patch properties",
+          "type": "object",
+          "x-nullable": true,
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "identity": {
+          "description": "The managed identity for the DigitalTwinsInstance.",
+          "x-nullable": true,
+          "$ref": "#/definitions/DigitalTwinsIdentity"
+        },
+        "properties": {
+          "description": "Properties for the DigitalTwinsInstance.",
+          "x-nullable": true,
+          "$ref": "#/definitions/DigitalTwinsPatchProperties"
+        }
+      }
+    },
+    "DigitalTwinsResource": {
+      "description": "The common properties of a DigitalTwinsInstance.",
+      "properties": {
+        "id": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource identifier."
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource name.",
+          "pattern": "^(?!-)[A-Za-z0-9-]{3,63}(?<!-)$"
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource type."
+        },
+        "location": {
+          "type": "string",
+          "description": "The resource location.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "tags": {
+          "type": "object",
+          "x-nullable": true,
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The resource tags."
+        },
+        "identity": {
+          "description": "The managed identity for the DigitalTwinsInstance.",
+          "x-nullable": true,
+          "$ref": "#/definitions/DigitalTwinsIdentity"
+        },
+        "systemData": {
+          "description": "Metadata pertaining to creation and last modification of the DigitalTwinsInstance.",
+          "readOnly": true,
+          "$ref": "#/definitions/SystemData"
+        }
+      },
+      "x-ms-azure-resource": true,
+      "required": [
+        "location"
+      ]
+    },
+    "DigitalTwinsIdentity": {
+      "description": "The managed identity for the DigitalTwinsInstance.",
+      "properties": {
+        "type": {
+          "type": "string",
+          "enum": [
+            "None",
+            "SystemAssigned",
+            "UserAssigned",
+            "SystemAssigned,UserAssigned"
```

Comment:
>SystemAssigned,UserAssigned [](http://example.com/codeflow?start=13&length=27)

This means both user and system assigned? Curious if there's any thought on providing one single for this case

Reply:
This is following the contract set by the Managed Identity team and what other RPs have defined in their swagger for User Assigned Identities: https://dev.azure.com/msazure/One/_releaseProgress?_a=release-pipeline-progress&releaseId=7864354